frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -1454,6 +1455,10 @@ class Worksheet extends React.Component {
                             items[focus].mode !== 'table_block'
                         ) {
                             this.setFocus(focus >= 0 ? focus + 1 : 'end', 0);
+                        } else if (this.state.focusIndex === -1) {
+                            // if currently the virtual item is focused and a new bundle has been uploaded to the worksheet
```

Comment:
What does "the virtual item" mean? Maybe we can clarify this comment a bit.

Reply:
change to the `top of the worksheet`